perf.groovy

```diff
@@ -3,8 +3,7 @@
 
 // Import the utility functionality.
 
-import jobs.generation.ArchivalSettings;
-import jobs.generation.Utilities;
+import jobs.generation.*;
```

Comment:
Why `*` instead of being explicit? I'm never in favor of relying on `*` for anything that can impact the dependencies of a build...

Reply:
This is fine in this case, no worries about conflicting imports.

Reply:
CoreCLR does it in this way - I am following their lead: https://github.com/dotnet/coreclr/blob/master/perf.groovy

Reply:
Also: https://github.com/dotnet/coreclr/blob/master/netci.groovy

Reply:
Just because it doesn't cause conflicts today, doesn't mean it can't/won't in the future.

Having any kind of `*` that impacts your build is generally bad practice. It makes things non-deterministic and means that you don't have explicit control over what things are being brought in.

We have explicitly kept away from using `*` everywhere else because of this. (CC. @jaredpar)

Reply:
@tannergooding In groovy here, files are required to map 1:1 to class names, so I don't think this can happen.  Java will also error if any imported class would have the same name as an already existing class.